src/app/log-in-options/log-in-options.component.html

```diff
@@ -1,46 +1,46 @@
-<div class="d-flex align-items-center flex-column">
-  <button
-    data-control-name="log-in-cta"
-    data-control-login-option="metamask"
-    class="button button-large mb-20px bg-dark text-white"
-    (click)="navigateToMetamaskSignup()"
-    queryParamsHandling="merge"
-    *ngIf="globalVars.showMetamask()"
-  >
-    <img src="assets/metamask.png" height="32px" width="32px" class="mr-10px" />
-
-    <span class="font-weight-bold">Sign up / log in with MetaMask</span>
-  </button>
-  <button
-    data-control-name="log-in-cta"
-    data-control-login-option="google"
-    class="button button-large button-primary mb-20px"
-    (click)="launchGoogle()"
-    *ngIf="!globalVars.hideGoogle"
-  >
-    <img
-      src="assets/google_logo.svg"
-      height="32px"
-      width="32px"
-      class="mr-10px"
-    />
-    <span class="font-weight-bold">Sign up / log in with Google</span>
-  </button>
-  <button
-    data-control-name="log-in-cta"
-    data-control-login-option="sign-up-with-deso-seed"
-    class="button button-large button-secondary mb-20px"
-    [routerLink]="['/sign-up']"
-    queryParamsHandling="merge"
-  >
-    <span class="font-weight-normal">Sign up with DeSo seed</span>
-  </button>
-  <a
-    data-control-name="log-in-cta"
-    data-control-login-option="log-in-with-deso-seed"
-    class="link" [routerLink]="['/load-seed']" queryParamsHandling="merge"
-  >
-    <u>Log in with DeSo seed</u>
-  </a
-  >
+<div class="section--login">
+  <div class="section--login__common">
+    <button
+      data-control-name="log-in-cta"
+      data-control-login-option="metamask"
+      class="button--primary button--medium margin-bottom--medium"
+      (click)="navigateToMetamaskSignup()"
+      queryParamsHandling="merge"
+      *ngIf="globalVars.showMetamask()"
+    >
+      <img src="assets/metamask.png" width="32px" class="mr-10px" />
+      Sign up / Log in with MetaMask
+    </button>
+    <button
+      data-control-name="log-in-cta"
+      data-control-login-option="google"
+      class="button--primary button--medium margin-bottom--medium"
+      (click)="launchGoogle()"
+      *ngIf="!globalVars.hideGoogle"
+    >
+      <img src="assets/google_logo.svg" width="32px" />
+      Sign up / Log in with Google
+    </button>
+  </div>
+  <div class="section--login__advanced">
+    <div class="margin-bottom--medium">
+      For advanced users only:
+    </div>
+    <button
+      data-control-name="log-in-cta"
+      data-control-login-option="sign-up-with-deso-seed"
+      class="button--primary--outline button--medium margin-bottom--medium"
+      [routerLink]="['/sign-up']"
+      queryParamsHandling="merge"
+    >
+      Sign up with DeSo seed
+    </button>
+    <a
+      data-control-name="log-in-cta"
+      data-control-login-option="log-in-with-deso-seed"
+      class="text--small" [routerLink]="['/load-seed']" queryParamsHandling="merge"
```

Comment:
The load seed page has no padding on the sides. Let's address this in a separate PR.

Reply:
Yup, will do a mobile pass at a later point to get everything responsive with padding on the sides.